c/extras/aggregator.h

```diff
@@ -0,0 +1,41 @@
+// License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
license header truncated

Reply:
Yes, something happened to my editor when pasted. Fixed.